railties/lib/rails/commands/devcontainer/devcontainer_command.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require "rails/generators"
+require "rails/generators/rails/devcontainer/devcontainer_generator"
+
+module Rails
+  module Command
+    class DevcontainerCommand < Base # :nodoc:
```

Comment:
`rails g devcontainer` would work if you pass options to it, or get the default options. But the command is what knows about booting the application and looking at it to figure out what options are needed.

I thinking about how I could move that into the generator. I think we would need another option for the generator to tell it to generate based on the current application, rather than the passed options (or just the defaults). So I guess we could make that option `true` by default (otherwise user has to do `rails g devcontainer --infer-options`), and then we can set it to `false` when we call it in the application generator.

What do you think about that?

Reply:
Let's keep the command then